both/gamelogic.js

```diff
@@ -4,7 +4,8 @@ GameLogic = {
   DOWN: 2,
   LEFT: 3,
   TIMER: 30,
-  CARD_SLOTS: 5
+  CARD_SLOTS: 5,
+  MIN_PLAYERS: 1
```

Comment:
= the minimum number of players necessary to start a game. I set this to 1 for easier testing.